Aspire Budgeting/Facilities/UserManager.swift

```diff
@@ -41,7 +41,11 @@ protocol AspireNotificationCenter: AnyObject {
 
 extension NotificationCenter: AspireNotificationCenter {}
 
-final class UserManager<U: AspireUser>: NSObject, GIDSignInDelegate, ObservableObject {
+protocol AppUserManager {
+  func authenticateExternally()
+}
+
+final class UserManager<U: AspireUser>: NSObject, GIDSignInDelegate, ObservableObject, AppUserManager {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d909f41c73c54120b2fbb4e96bb79560130cb154/Aspire Budgeting/Facilities/UserManager.swift#L48"></span>Line should be 100 characters or less: currently 103 characters<br />
<code>line_length</code> <code>UserManager.swift:48</code></td>
    </tr>
  </tbody>
</table>